receiver/filestatsreceiver/metadata.yaml

```diff
@@ -0,0 +1,52 @@
+name: filestatsreceiver
+
+status:
+  type: receiver
+  stability: development
+  pipelines: [metrics]
+  distributions: [contrib]
+
+
+resource_attributes:
+  file.name:
+    description: The name of the file
+    enabled: true
+    type: string
+  file.path:
+    description: The absolute path of the file
+    enabled: false
+    type: string
+
+attributes:
+  file.permissions:
+    description: the permissions associated with the file, using an octal format.
+    type: string
+
+metrics:
+  file.mtime:
+    description: Timestamp of the last modification of the file or folder, in seconds since Epoch.
+    enabled: true
+    gauge:
```

Comment:
It might be worth checking what the specification says, but these timestamp metrics seems like cumulative monotonic counters to me

Reply:
I looked in https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/README.md and didn't see anything for files.
It's also possible for someone to set the mtime to an arbitrary time: https://superuser.com/questions/604443/manually-update-the-mtime-of-a-file-in-unix

Reply:
I mean type of the metric should be driven by its nature. I believe elapsed time is a monotonic counter, not a gauge. See all other `...time` metrics in https://github.com/open-telemetry/opentelemetry-specification/blob/294f58bac25b444e0c14ee13f39eb9ffb5238dea/specification/metrics/semantic_conventions/system-metrics.md as example

Reply:
Sorry, I just realized that it's a static timestamp. I thought it was elapsed time since creation/modification/access. Maybe it makes sense to keep it gauge in that case, but do you think it's more user-friendly than elapsed time? I saw several uptime-like metrics in contrib, but no static timetamps like this one.

Reply:
We can use that approach instead, sure. The use case is fulfilled either way.